config.cson

```diff
@@ -4116,7 +4116,7 @@ fileIcons:
 		icon: "webpack"
 		priority: 2
 		match: [
-			[/webpack\.conf(ig)?\./i, "medium-blue"]
+			[/webpack(\.\w+)?\.conf(ig)?\./i, "medium-blue"]
```

Comment:
Actually, you may as well make that `(\.\w+)*` instead, so something like `webpack.foo.bar.config.js` will match too.